tests/test/scheduler/test_mpi_world.cpp

```diff
@@ -471,16 +471,6 @@ TEST_CASE("Test send across hosts", "[mpi]")
     remoteWorld.send(
       rankA, rankB, BYTES(messageData.data()), MPI_INT, messageData.size());
 
-    SECTION("Check queueing")
```

Comment:
Issuing a `faabric::MpiWorld::send` does no longer wait for completion. Thus, checking the queues right after we send a message will most likely fail.

The workaround is to force a purely-synchronous message. This behaviour, together with a test for correct enqueueing, is already provided in `[Test sending MPI message] [scheduler]`. 

I can also workaround a test for the queues in this file, but it wouldn't use the `MpiWorld` 's API.